flake.nix

```diff
@@ -21,6 +21,7 @@
           yq
           kyverno-chainsaw
         ];
+        CGO_ENABLED = "0";
```

Comment:
This was added to make delve work with nix. Before adding this, we were using CGO locally, but not in the docker image that publish. This makes the development environment similar to what we ship and also allows us not to link against nix's C libs, so that we can run the debugger in vscode.